internal/manifests/var.go

```diff
@@ -38,12 +38,40 @@ const (
 	LabelQueryFrontendComponent string = "query-frontend"
 )
 
+var (
+	defaultConfigMapMode = int32(420)
+	volumeFileSystemMode = corev1.PersistentVolumeFilesystem
+)
+
+func merge(ms ...map[string]string) map[string]string {
+	res := map[string]string{}
+	for _, m := range ms {
+	srcMap:
+		for k, v := range m {
+			// Check if (k,v) was added before:
+			for _, v2 := range res {
+				if v == v2 {
+					continue srcMap
+				}
+			}
+			res[k] = v
+		}
+	}
+	return res
+}
```

Comment:
What is this function doing differently than `mergo.Merge`?

Reply:
Code changed to use mergo.Merge